components/launcher-protocol/protocols/launcher.proto

```diff
@@ -59,3 +59,12 @@ message PidIs {
   // way to evolve that?
   optional uint32 pid = 1;
 }
+
+// Query the Launcher to determine the launcher's version
+message Version {}
+
+// The response that corresponds to `Version` and contains the
+// version number.
+message VersionNumber {
+  optional uint32 version = 1;
```

Comment:
Could this be `required`?

Reply:
maybe? I was trying to conform to the convention used for all the other messages which all specify `optional`. I just assume there is a reason for that but I'm not certain what it is.

Reply:
I'm not sure on the original intent, but in proto-land there is a strong bias against required fields. So far so that they were removed in proto3.  The reason that is typically given is that `required` fields make it hard to make non-breaking changes to the message down the road.